packages-exp/performance-exp/index.ts

```diff
@@ -17,68 +17,32 @@
 
 import '@firebase/installations';
 
-import firebase from '@firebase/app';
-import { FirebaseApp, FirebaseNamespace } from '@firebase/app-types';
+import { FirebaseApp } from '@firebase/app-types';
```

Comment:
app-types-exp

Reply:
Do we want to migrate these now, or once all other packages have moved to exp? So far I only updated the internal references to exp, not for our dependencies.

Reply:
Since there are some missing packages right now, this may not be viable. Once the rest of our dependencies are in -exp we can make this change.

Reply:
You should be able to use `@firebase/app-types-exp`. I think the problem is you are not using the component framework which we will continue to use in vNext.

You should still register Perf with a factory method, like you are doing currently. It would look something like:
```
import { _registerComponent, _getProvider } from '@firebase/app-exp';

_registerComponent(
  new Component(
      'performance',
      container => {
        const app = container.getProvider('app-exp').getImmediate();
        const installations = container
          .getProvider('installations')
          .getImmediate();

        return factoryMethod(app, installations);
      },
      ComponentType.PUBLIC
));

export getPerformance(): Performance {
  const provider: Provider<'performance'> = _getProvider(
    app,
    'performance'
  );
  const perfInstance = provider.getImmediate();
  return perfInstance;
};
```

Reply:
Thanks for the information! I will do this in a follow up PR since I've got quite a few changes built on top of this PR and want to avoid the merge conflict (and since this is not usable yet, there's no danger of things not working).